### PR TITLE
Update to ember 3.28

### PR DESCRIPTION
**Pull request by Angel-M-R**

### CODE REVIEW

**Review by adrigzr — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -37,7 +39,7 @@ Inject the service in your app:
 
 ```javascript
 export default Component.extend({
```
Comment (josex2r) [inline README.md:41]: Upgrade this to Octane:

```js
export default class MyClass {
    @service network;
}
```

> Ensure that all examples are in Octane way please.

File: addon/services/network.js [hunk 2]
```diff
@@ -6,19 +6,19 @@ import { cancel, later } from '@ember/runloop';
...
+	lastReconnectStatus = 0;
 
-	@tracked _timer;
-	@tracked _times;
-	@tracked _timestamp;
-	@tracked _state;
-	@tracked _controllers;
-	@tracked _config;
-	@tracked _nextDelay;
+	_timer;
```
Comment (josex2r) [inline addon/services/network.js:15]: We moved from properties to tracked almost 1 year ago, do not reverse the changes again.

- [ ] Use tracked properties

**Review by josex2r — CHANGES_REQUESTED** [review]

**Review by josex2r — DISMISSED** [review]

**Review by adrigzr — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -8,90 +8,120 @@
...
-  "config": {
-    "commitizen": {
-      "path": "./node_modules/cz-conventional-changelog"
-    }
+    "@glimmer/tracking": "^1.0.4",
+    "ember-cli-babel": "^7.26.6",
+    "ember-cli-htmlbars": "^5.7.1",
+    "ember-task-scheduler": "^2.1.0",
+    "ember-transition-end": "^2.0.0",
+    "lint-staged": "^12.1.2"
```
Comment (adrigzr) [inline package.json:53]: Esto es una dep de desarrollo.

**Review by adrigzr — CHANGES_REQUESTED** [review]
Cambia el package.json

File: package.json [hunk 4]
```diff
@@ -8,90 +8,120 @@
...
-    "babel-plugin-ember-modules-api-polyfill": "2.13.0"
-  },
-  "config": {
-    "commitizen": {
-      "path": "./node_modules/cz-conventional-changelog"
-    }
+    "@glimmer/tracking": "^1.0.4",
+    "ember-cli-babel": "^7.26.6",
+    "ember-cli-htmlbars": "^5.7.1",
+    "ember-task-scheduler": "^2.1.0",
```
Comment (adrigzr) [inline package.json:51]: Esta dep no veo que se use.

File: package.json [hunk 5]
```diff
@@ -8,90 +8,120 @@
...
-    "ember-cli-babel": "~7.19.0",
-    "babel-plugin-ember-modules-api-polyfill": "2.13.0"
-  },
-  "config": {
-    "commitizen": {
-      "path": "./node_modules/cz-conventional-changelog"
-    }
+    "@glimmer/tracking": "^1.0.4",
+    "ember-cli-babel": "^7.26.6",
+    "ember-cli-htmlbars": "^5.7.1",
```
Comment (adrigzr) [inline package.json:50]: Esta dep va en dev.

File: package.json [hunk 6]
```diff
@@ -8,90 +8,120 @@
...
+    "ember-task-scheduler": "^2.1.0",
+    "ember-transition-end": "^2.0.0",
```
Comment (adrigzr) [inline package.json:52]: Esta dep no veo que se use, de dónde lo has copiado?

**Review by adrigzr — CHANGES_REQUESTED** [review]

**Review by adrigzr — APPROVED** [review]

**Pull request merged**